external-auth/manifests/selkies-opa.yaml

```diff
@@ -172,3 +172,20 @@ spec:
         image: openpolicyagent/kube-mgmt:0.12.1
         args:
           - --enable-data
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  labels:
+    app.kubernetes.io/name: broker
+  name: deny-all
```

Comment:
This name should be something like `allow-istio`

Reply:
Done